Detectors/TPC/workflow/src/ProcessingHelpers.cxx

```diff
@@ -61,3 +63,11 @@ uint64_t processing_helpers::getCreationTime(o2::framework::ProcessingContext& p
 {
   return DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
 }
+
+uint64_t processing_helpers::getTimeStamp(o2::framework::ProcessingContext& pc, o2::ccdb::BasicCCDBManager& ccdbManager)
+{
+  const auto tfOrbitFirst = DataRefUtils::getHeader<o2::header::DataHeader*>(pc.inputs().getFirstValid(true))->firstTForbit;
+  const auto* tv = ccdbManager.getForTimeStamp<std::vector<Long64_t>>("CTP/Calib/OrbitReset", processing_helpers::getCreationTime(pc));
```

Comment:
I think the `CTP/Calib/OrbitReset` should also come from using something like
```c++
inputSpecs.emplace_back("tpcpadmap", "CTP", "OrbitReset", 0, Lifetime::Condition, ccdbParamSpec("CTP/Calib/OrbitReset"));
```
and then pass the `tv` from the workflow to this function. Using ccdb manager directly from a workflow should not be done any longer.
Perhaps you can check how it is done in the ccdb access in the framework. I could imagine that `CTP/Calib/OrbitReset` is already available default. But I don't know for sure.

Reply:
Thanks for pointing that out. I just fixed it by loading the OrbitReset from the CCDB directly via  setting it as an input spec.